### DISCUSSION TITLE
The ability to choose osu!cookie as it is done with the intro sequence

### DISCUSSION THREAD

**Discussion by MrLimeick** [ppy/osu#27848] [Ideas] [answered]
It would be fun to see the 2007 osu!cookie in lazer, plus I miss the old lazer icon a bit.
Sorry for the mistakes, I'm using a translator :(

**Comment by Joehuu** [comment, marked as answer]
Hello, thanks for opening a feature request. This is a duplicate of https://github.com/ppy/osu/discussions/26623 (it spans more, but has the old logo idea) so please upvote that original discussion to show your support!
